src/main/java/net/sf/jabref/bst/BibtexCaseChanger.java

```diff
@@ -129,17 +134,17 @@ private String doChangeCase(String s, FORMAT_MODE format) {
         return sb.toString();
     }
 
-    private int decrBraceLevel(String string, int braceLevel) {
-        if (braceLevel == 0) {
+    private int decrBraceLevel(String string, int bLevel) {
+        if (bLevel == 0) {
```

Comment:
This is one of the actual cases where Eclipse warns. braceLevel is a variable in the BibTexCaseChanger class, so using the same name for a method argument seems like a bad idea. I can of course give it a longer name rather than abbreviate it. (Using the same name seems more incorrect here than in the earlier cases we've discussed: constructors and initfunctions.)